README.md

```diff
@@ -1,6 +1,6 @@
 # HMPPS Interventions Service
 
-[![API docs](https://img.shields.io/badge/API_docs-view-85EA2D.svg?logo=swagger)](https://hmpps-interventions-ui-dev.apps.live-1.cloud-platform.service.justice.gov.uk/swagger-ui.html)
+[![API docs](https://img.shields.io/badge/API_docs-view-85EA2D.svg?logo=swagger)](https://hmpps-interventions-service-dev.apps.live-1.cloud-platform.service.justice.gov.uk/swagger-ui/index.html?configUrl=/v3/api-docs/swagger-config)
```

Comment:
this should be ```https://hmpps-interventions-service-dev.apps.live-1.cloud-platform.service.justice.gov.uk/swagger-ui.html``` the internal redirect could change

Reply:
done in 63edafc6c7fba9cc7e37cf2e5a25ef4f1bea80f4, thanks!